### DISCUSSION TITLE
Show order [management]: Adapt column headings? + append supplier order unit for all articles?

### DISCUSSION THREAD

**Discussion by twothreenine** [foodcoops/foodsoft#1237] [Decisions]
Column headings:
"Members" is confusing and was never translated. I suggest changing it to "Total order", or whatever term we agree on in foodcoops/foodsoft#1097.
Same for "Full units" -> "Order result"?

Then for some articles, the supplier order unit isn't shown in the far right column:
- In case of beer, you wouldn't know if it refers to `20.0 l` or `20.0 x Bottle (0.5 l)`.
- In case of potatoes, we need it since it differs from the group order unit.
- For carrots & oyster mushrooms (supplier order unit == group order unit) we don't really need it, I'm unsure if it should be appended for the sake of completeness or omitted to underline it's the same as the group order unit.

![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/80396f28-5e17-4eeb-af0d-edf799c33f88)

This also applies to the receive menu:
![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/e3053009-affa-4317-b9e1-bcb09cd5d71d)

**Comment by twothreenine** [comment]
I thought about the terminology in the column headings across various views, also in regards to [upstream/#1016](https://github.com/foodcoops/foodsoft/issues/1016).
My key thoughts:
- there should be a consistent terminology in the frontend
- some headings should depend on the order's state

Here are my notes, which go beyond this issue and also beyond the aim of this fork:
[Hackathon received etc columns.xlsx](https://github.com/foodcoopsat/foodsoft_hackathon/files/15236863/Hackathon.received.etc.columns.xlsx)

**Comment by lentschi** [comment]
The issue with the missing units is fixed by https://github.com/foodcoopsat/foodsoft_hackathon/commit/b38dfb0c25be9ef3f1a6eee7d17615043c0ce9bf, the issue with the column renaming/ordering remains, but is not directly related to the fork, so I'll move it to the post-merge milestone.
